zcash_client_sqlite/src/wallet.rs

```diff
@@ -978,14 +919,7 @@ pub fn delete_utxos_above<'a, P: consensus::Parameters>(
     taddr: &TransparentAddress,
     height: BlockHeight,
 ) -> Result<usize, SqliteClientError> {
-    let sql_args: &[(&str, &dyn ToSql)] = &[
-        (":address", &taddr.encode(&stmts.wallet_db.params)),
-        (":above_height", &u32::from(height)),
-    ];
-
-    let rows = stmts.stmt_delete_utxos.execute_named(sql_args)?;
-
-    Ok(rows)
+    stmts.stmt_delete_utxos(taddr, height)
```

Comment:
some of these are now so trivial as to almost not need to exist, but I guess that it's fine

Reply:
Yep, these amount to a move in their cases. But they are all in the public API (deprecated), so I left them alone for now (we can decide what to do with them separately).